__tests__/database/migration/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('TESTING migration file', () => {
     expect(DROP_ALL_DB).toBeFalsy();
   });
 
-  it('TEST should expect TARGET_MIGRATION_VERSION to be 17', () => {
-    expect(TARGET_MIGRATION_VERSION).toBe(17);
+  it('TEST should expect TARGET_MIGRATION_VERSION to be 21', () => {
+    expect(TARGET_MIGRATION_VERSION).toBe(21);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case has been updated to reflect the new `TARGET_MIGRATION_VERSION` value of 21. This change is consistent with the PR description stating that the number of migrations has increased. However, it's important to ensure that all necessary migration scripts for versions 18 through 21 have been properly implemented and tested.
```diff
-    expect(TARGET_MIGRATION_VERSION).toBe(17);
+    expect(TARGET_MIGRATION_VERSION).toBe(21);
```
If these migration scripts are not present or have not been thoroughly tested, this could lead to potential issues when migrating from an older database version to the latest one.


<!-- This is an auto-generated comment by OSS CodeRabbit -->